api/contrib/envoy/extensions/dlb/v3alpha/dlb.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.extensions.dlb.v3alpha;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.dlb.v3alpha";
+option java_outer_classname = "DlbProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/dlb/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Dlb connection balancer configuration]
+// DLB :ref:`configuration overview <config_connection_balance_dlb>`.
+// [#extension: envoy.network.connection_balance.dlb]
+
+// The Dlb is a hardware managed system of queues and arbiters connecting producers and consumers. It is a PCIE device
+// in the CPU package. It interacts with software running on cores and potentially other devices. The Dlb implements the
+// load balancing features outlined earlier, including the following:
+// -  Lock-free multi-producer/multi-consumer operation.
```

Comment:
this list is collapsing - i think it needs a space above

Reply:
Add a new line above.